data/remnant/remnant 2 cognizance.txt

```diff
@@ -384,7 +384,7 @@ mission "Remnant: Cognizance 9"
 				`	"What do you mean?"`
 				`	"Let's get on with it."`
 					goto skipinfo
-			`	"Well, this place is fairly old. It was built shortly after we learned how to build Starlings - which is where we learned of the troubles of extending the cloaking field beyond small ships. I don't know the mechanics behind it, but it has taken close to two centuries of study and development for cloaks to be feasible on large ships. We had plenty of Puffins back then, and the techniques for building high-pressure systems were well understood. The Alphas hadn't shown any interest in gas giants or 'gaslined ships' as we call them, so having somewhere like this to hide in makes sense. Best place to hide is somewhere no one will look, right?" With that, they nod to themself, climbs to his feet, and heads towards the suit locker.`
+			`	"Well, this place is fairly old. It was built shortly after we learned how to build Starlings - which is where we learned of the troubles of extending the cloaking field beyond small ships. I don't know the mechanics behind it, but it has taken close to two centuries of study and development for cloaks to be feasible on large ships. We had plenty of Puffins back then, and the techniques for building high-pressure systems were well understood. The Alphas hadn't shown any interest in gas giants or 'gaslined ships' as we call them, so having somewhere like this to hide in makes sense. Best place to hide is somewhere no one will look, right?" With that, they nod to themself, climb to their feet, and head towards the suit locker.`
```

Comment:
I know this is the old phrase, but while we are here, I have a question about `we learned how to build Starlings`, this implies that it was taught. Was the knowledge given to them by someone/thing? Did they find instructions? If it is their own work and design, I would expect `figured out`, not `learned`.

Reply:
I personally interpret it as ambiguous: it wasn't entirely their own creation, but it wasn't taught per se either. Something in the middle.

Reply:
I agree that 'learned' sounds like they're being taught, though. If you want to be ambiguous, then maybe something like 'discerned' would be better?

Reply:
I'll think about it, but neither "figured out" nor "discerned" really sit well with me; so I would prefer to leave that for another time.

Reply:
`shortly after we developed Starlings`
`shortly after we began building Starlings` - not a fan of this one since it follows `built` with `building` so soon.
`shortly after we built the first Starlings`
`shortly after we built our first Starlings`

The use of `learned` again a few words later doesn't really help, but the meaning there is fine, those issues they encountered were not their own invention, they exist regardless of if the Remnant know of them.

Reply:
`shortly after the Starling's inception`, perhaps? Implying that it _began_ around that time, but not necessarily that the Starling design was of remnant origin in its entirety...

Reply:
I'll continue to think about this, but I'm still not even convinced it needs to be changed, so I'm resolving this so it doesn't hold up the rest of these fixes.